docs/chapters/control/content.ru-RU.md

```diff
@@ -0,0 +1,81 @@
+# Контроль кривых Безье
+
+Кривые Безье, как и все отрезки кривых, являются интерполярными функциями. Это значит, что они имеют заданный набор контрольных точек и генерируют значения где-то "между" ними. (Одним следствием этого есть то, что невозможно сгенерировать точки вне этого "каркаса". Полезная информация!). На практике мы можем проиллюстрировать влияние каждой контрольной точки на течение кривой по отдельности и проследить какие из точек имеют наибольший эффект в тех и иных частях кривой.
+
+Следующие зарисовки демонстрируют интерполяции функций для квадратной и кубической кривых, где "S" представляет степень влияния контрольной точки на положение участка кривой по ходу развития t. Потяните ползунки, чтобы отследить проценты интерполяций каждой контрольной точки на конкретном <i>t</i>.
+
+<div class="figure">
+<graphics-element title="Квадратная интерполяция" src="./lerp.js" data-degree="3">
+  <input type="range" min="0" max="1" step="0.01" value="0" class="slide-control">
+</graphics-element>
+<graphics-element title="Кубическая интерполяция" src="./lerp.js" data-degree="4">
+  <input type="range" min="0" max="1" step="0.01" value="0" class="slide-control">
+</graphics-element>
+<graphics-element title="Интерполяция кривой 15-го порядка" src="./lerp.js" data-degree="15">
+  <input type="range" min="0" max="1" step="0.01" value="0" class="slide-control">
+</graphics-element>
+</div>
+
+Также представлена интерполяция кривой Безье 15<sup>го</sup> порядка. Как вы можете видеть, начальная и конечная точки влияют на положение кривой значительно больше, чем любая другая контрольная точка.
+
+Желая изменить кривую, нам нужно изменить "вес"-а (* коофициенты) для каждой точки, по сути меняя интерполяцию. Имплементация оного предельно логична: нужно просто перемножить каждую точку на значение, меняющее ее 'силу'. Эти значения, по конвенции называют "весом" и мы можем учесть их запись в нашей оригинальной функции Безье:
+
+\[
+  Bézier(n,t) = \sum_{i=0}^{n}
+                \underset{binomial~term}{\underbrace{\binom{n}{i}}}
+                \cdot\
+                \underset{polynomial~term}{\underbrace{(1-t)^{n-i} \cdot t^{i}}}
+                \cdot\
+                \underset{weight}{\underbrace{w_i}}
+\]
```

Comment:
the `binomial~term`, `polynomial~term`, and `weight` under the undersets can also be translated (the `~` is a special LaTeX space in maths mode)